utxo/utxo_meta.go

```diff
@@ -363,6 +363,16 @@ func (uv *UtxoVM) LoadGasPrice() (*pb.GasPrice, error) {
 		if cpuRate < 0 || memRate < 0 || diskRate < 0 || xfeeRate < 0 {
 			return nil, ErrProposalParamsIsNegativeNumber
 		}
+		// To be compatible with the old version v3.3
+		// If GasPrice configuration is missing or value euqals 0, support a default value
+		if cpuRate == 0 && memRate == 0 && diskRate == 0 && xfeeRate == 0 {
```

Comment:
What if the user DO NOT want to use gas? Can I manually set all rate to 0 in genesis config?

Reply:
Agreed to solve this issue later in non-gas improvement